homeassistant/components/climate/strings.json

```diff
@@ -90,10 +90,10 @@
           "state": {
             "none": "None",
             "eco": "Eco",
-            "away": "Away",
+            "away": "[%key:common::state::not_home%]",
```

Comment:
This seems incorrect to me. While `not_home` is used as `Away` for device trackers, it is referring to the `not_home` state. This is the `away` state. Those are different states.

Reply:
I was not aware of that. Thanks, reverted.